bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -164,12 +167,16 @@ std::error_code BoltAddressTranslation::parse(StringRef Buf) {
 
     LLVM_DEBUG(dbgs() << "Parsing " << NumEntries << " entries for 0x"
                       << Twine::utohexstr(Address) << "\n");
+    uint64_t InputOffset = 0, OutputOffset = 0;
```

Comment:
Maybe I'm missing something, but I don't understand why are we naming this as Offset. Even in BAT.md it is listed as 'address' instead of offset.